client/gutenberg/editor/controller.js

```diff
@@ -50,51 +45,6 @@ function getPostID( context ) {
 	return parseInt( context.params.post, 10 );
 }
 
-function waitForSiteIdAndSelectedEditor( context ) {
-	return new Promise( ( resolve ) => {
-		const unsubscribe = context.store.subscribe( () => {
-			const state = context.store.getState();
-			const siteId = getSelectedSiteId( state );
-			if ( ! siteId ) {
-				return;
-			}
-			const selectedEditor = getSelectedEditor( state, siteId );
```

Comment:
I think we can probably remove the `getSelectedEditor` selector entirely from the codebase too, along with the `requestSelectedEditor` action. We don't care about the selected editor anymore, we only support Gutenberg now.

But it's fine to do this in a follow-up.

Reply:
Maybe I went too far @mmtr :D